Doc/c-api/structures.rst

```diff
@@ -147,23 +147,56 @@ Implementing functions and methods
    value of the function as exposed in Python.  The function must return a new
    reference.
 
+   The function signature is::
+
+      PyObject *PyCFunction(PyObject *self,
+                            PyObject *const *args);
```

Comment:
Inconsistent argument type compared to `typedef PyObject *(*PyCFunction)(PyObject *, PyObject *);`
https://github.com/python/cpython/blob/master/Include/methodobject.h#L18